wifinina/wifinina.go

```diff
@@ -296,8 +296,8 @@ func (d *Device) StartClient(addr uint32, port uint16, sock uint8, mode uint8) e
 		println("[StartClient] called StartClient()\r")
 		fmt.Printf("[StartClient] addr: % 02X, port: %d, sock: %d\r\n", addr, port, sock)
 	}
-	if err := d.waitForSlaveSelect(); err != nil {
-		d.spiSlaveDeselect()
+	if err := d.waitForDeviceSelect(); err != nil {
+		d.spiDeviceDeselect()
```

Comment:
The new name, according to https://www.oshwa.org/a-resolution-to-redefine-spi-signal-names, is "chip select". In fact, "chip select" (or CS) seems to be quite common already. I would suggest changing to that instead of "device select".

Reply:
Done.